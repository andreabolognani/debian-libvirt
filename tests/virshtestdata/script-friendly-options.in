echo "Following script-friendly outputs of commands must not change"
list --uuid;
list --uuid --all;
list --name;
list --name --all;
list --uuid --name --all;
domstate fv0
domstate fv0 --reason
domname ef861801-45b9-11cb-88e3-afbfe5370493
domid fv0
domstats fv0
