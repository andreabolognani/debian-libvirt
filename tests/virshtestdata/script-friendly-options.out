Following script-friendly outputs of commands must not change
4dea22b3-1d52-d8f3-2516-782e98ab3fa0
ef861801-45b9-11cb-88e3-afbfe5370493
08721f99-3d1d-4aec-96eb-97803297bb36

4dea22b3-1d52-d8f3-2516-782e98ab3fa0
ef861801-45b9-11cb-88e3-afbfe5370493
08721f99-3d1d-4aec-96eb-97803297bb36

fv0
fc4
fc5

fv0
fc4
fc5

4dea22b3-1d52-d8f3-2516-782e98ab3fa0 fv0
ef861801-45b9-11cb-88e3-afbfe5370493 fc4
08721f99-3d1d-4aec-96eb-97803297bb36 fc5

running

running (unknown)

fc4

1

Domain: 'fv0'
  state.state=1
  state.reason=0

